jvm/src/Language/Java.hs

```diff
@@ -327,7 +327,9 @@ getStaticField
 {-# INLINE getStaticField #-}
 getStaticField cname fname = do
   let retsing = sing :: Sing ty
-      klass = unsafeDupablePerformIO $ findClass (referenceTypeName (SClass (JNI.toChars cname)))
+      klass = unsafeDupablePerformIO $
+                findClass (referenceTypeName (SClass (JNI.toChars cname)))
```

Comment:
Oops, I guess I misunderstood what was going on here. I thought `newGlobalRef` was for tracking calls to `new`.